nselib/http.lua

```diff
@@ -2804,7 +2711,19 @@ function save_path(host, port, path, status, links_to, linked_from, contenttype)
   end
 
   -- Parse the path
-  local parsed = parse_url(path)
+  local parsed = url.parse(path)
+
+  -- contains both query and fragment
+  parsed.raw_querystring = parsed.query
+  if parsed.fragment then
+    parsed.raw_querystring = ( parsed.raw_querystring or "" ) .. "#" .. parsed.fragment
+  end
+
+  if parsed.raw_querystring then
+    parsed.path_query = parsed.path .. "?" .. parsed['raw_querystring']
```

Comment:
Please consider using consistent coding convention to maintain better clarity.
Up to this point the code was `parsed.raw_querystring` but here it is `parsed['raw_querystring']`